backend/session/direct.c

```diff
@@ -76,30 +77,39 @@ static void direct_session_close(struct wlr_session *base, int fd) {
 
 static bool direct_change_vt(struct wlr_session *base, unsigned vt) {
 	struct direct_session *session = wl_container_of(base, session, base);
+
+	// Only seat0 has VTs associated with it
+	if (strcmp(session->base.seat, "seat0") == 0) {
```

Comment:
So this _doesn't_ switch VTs if the user is on `seat0`?

Reply:
That was a silly mistake.